src/pages/registration/DescriptionPanel.tsx

```diff
@@ -84,20 +84,53 @@ export const DescriptionPanel = () => {
           )}
         </Field>
       </Box>
-      <Field name={DescriptionFieldNames.Abstract}>
-        {({ field }: FieldProps<string>) => (
-          <TextField
-            {...field}
-            value={field.value ?? ''}
-            data-testid={dataTestId.registrationWizard.description.abstractField}
-            variant="filled"
-            fullWidth
-            multiline
-            rows="4"
-            label={t('registration.description.abstract')}
-          />
-        )}
-      </Field>
+      <Box
+        sx={{
+          display: 'grid',
+          rowGap: '1rem',
+          columnGap: '0.5rem',
+          gridTemplateColumns: '1fr auto',
```

Comment:
kanskje sentrer knappen?
```suggestion
          gridTemplateColumns: '1fr auto',
          alignItems: 'center'
```

Reply:
Går for 'top', da det er det som er vist i skissene.